### ISSUE TITLE
Support ISBN 10 and ISBN13

### ISSUE THREAD

**Issue by phoenisx** [brianvoe/gofakeit#367]
Is it possible to support something similar to https://github.com/faker-js/faker/issues/2196?
Ref: https://fakerjs.dev/api/commerce#isbn

I am learning Go, and would be happy to contribute if we agree.
Let me know if this needs to be part of a Book instance or if we need to create a separate `Commerce` struct to manage this resource?

**Comment by brianvoe** [comment]
Yes go for it! I would have no problem adding ISBN. Plus it will be added to gofakeit.com

Since your new Ill give you a little checklist for pr's

1. Function - Add it in the right file
2. 3 functions - Global, Struct and private
3. Lookup - each file at the end has a lookup so it can be used in other places
4. Tests - each file also has a corresponding tests file. make sure to add your stuff there too

Thanks again and good luck!
